code/datums/mutantraces.dm

```diff
@@ -1104,13 +1104,15 @@ TYPEINFO(/datum/mutantrace)
 	decomposes = FALSE
 	race_mutation = /datum/bioEffect/mutantrace/skeleton
 	dna_mutagen_banned = FALSE
-	var/obj/item/organ/head/head
+	var/obj/item/organ/head/head_tracker
 
 	New(var/mob/living/carbon/human/M)
 		..()
 		if(ishuman(M))
 			M.mob_flags |= IS_BONEY
 			M.blood_id = "calcium"
+			head_tracker = M.organHolder.head
+			head_tracker.linked_human = M
```

Comment:
1114 is covered by the `set_head` proc, probably move 1115 to under the proc as well.